cocos/ui/progress-bar.ts

```diff
@@ -287,7 +287,7 @@ export class ProgressBar extends Component {
             const entTrans = entity._uiProps.uiTransformComp!;
             const entityAnchorPoint = entTrans.anchorPoint;
             const entitySize = entTrans.contentSize;
-            const entityPosition = entity.getPosition();
+            const entityPosition = entity.position;
```

Comment:
I think it is better also move this sentence just before its usage
```
entity.setPosition(entityPosition.x + finalPosition.x, entityPosition.y + finalPosition.y, entityPosition.z);
```

Reply:
I move it. And fix the finalPosition variable means.